### DISCUSSION TITLE
How you can help

### DISCUSSION THREAD

**Discussion by 152334H** [152334H/DL-Art-School#4] [General]
Thank you for checking out this page!

This project is very new and will probably change substantially really quickly, but right now, the most pressing problems I can think of are,

1. [Run training experiments](https://github.com/152334H/DL-Art-School/issues/1)! I have no idea what the optimal parameters are yet, and it'll be easier to figure out with more experimental data.
2. Build better training interfaces. We [could use a colab notebook](https://github.com/152334H/DL-Art-School/issues/2), but anything that improves the current admittedly janky state of training could help.
3. See the rest of the [issues](https://github.com/152334H/DL-Art-School/issues/) page. Problems that I would really like help from the public with are tagged with `good first issue` and `help wanted`, but any issue can be tackled.

**Comment by rikabi89** [comment]
I have no programming/coding knowledge but like you guys I am trying to find out what is the "optimal" training settings and will share anything note worthy. Is it worth setting up a discord?

**Reply by 8tm** [reply]
I can help with basics - not with AI but with functional issues like saving space, fixing colab template, create additional options.
Discord can be a good place to start to discuss some ideas.

**Reply by rikabi89** [reply]
https://discord.gg/gzTxHe3auT

I setup a channel on my discord. We can use this unless the owner wishes to setup an official channel. In the meantime everyone is welcome to share ideas.

**Comment by TEJASAMA-007** [comment]
Hello, I have finetuned the ARmodel for 1000 iterations. Now, how can I make an inference to that checkpoint?
